.npmignore

```diff
@@ -17,11 +17,11 @@ typings
 
```

Comment:
If you want, I will create a feature-brach only containing the fix-commit and create a new pull request with this branch. Please let me know.